src/pages/ManageProfile.tsx

```diff
@@ -12,7 +14,11 @@ import Tab from "react-bootstrap/Tab";
 
 const ManageProfilePage: React.FunctionComponent<
   IPage & RouteComponentProps<any>
-> = () => {
+> = (props) => {
+  useEffect(() => {
+    logging.info(`Loading ${props.name}`);
+  }, [props.name]);
```

Comment:
FWIW, you might be able to use the React Profiler to get the same insight that these logs are giving you.

Reply:
Thanks for the suggestion! Implemented here 9c12d8b